app/Models/User.php

```diff
@@ -47,6 +47,7 @@ class User extends Authenticatable implements WebAuthnAuthenticatable
 		'username',
 		'password',
 		'email',
+		'fullname',
```

Comment:
Please add the new property to the PHPdoc of the class, too.

Moreover, personally, I would like the denomination `display_name` better. This is also the name of an LDAP attribute and reflects more precisely the purpose of the attribute.

Reply:
You can spend the rest of your life improving all the names. I left that game long ago.
fullname does not belong to the domain of LADP. There it is called cn (common name).

Lychee uses an numerical id, to internally separate users and as a 'display name', the user name is used.
This is a somewhat limited concept. 

I am fine with the numerical id for handling things internally. Then we need a login name which is called uid in the context of LDAP (at least most of the time). On top we need a nice display name,

So, now you can define names for this like: (id, username, fullname) or (id,uid,cn) or (user_id, user_name, display_name).

I leave it up to you...